distribution/lib/Standard/Base/0.0.0-dev/src/System/File_Format.enso

```diff
@@ -117,73 +112,70 @@ type File_Format
 ## A file format for plain text files.
 type Plain_Text_Format
     ## A file format for plain text files with the specified encoding.
+
+       Arguments:
+       - encoding: The encoding to use.
+         If the encoding is set to `Infer`, it will be inferred from metadata if
+         available, falling back to UTF-8. For writing, `Infer` will also imply
+         the UTF-8 encoding as the default.
     @encoding Encoding.default_widget
-    Plain_Text (encoding:Encoding=Encoding.utf_8)
+    Plain_Text (encoding:Encoding|Infer = Encoding.utf_8)
```

Comment:
Shouldn't we default to `Infer` here?

Reply:
Sure, I was thinking about it, but didn't want to change the default unconsulted.

I'll amend it.